ui/app/[locale]/(authorized)/kafka/[kafkaId]/consumer-groups/[groupId]/reset-offset/Dryrun.tsx

```diff
@@ -100,7 +101,7 @@ export function Dryrun({
           </FlexItem>
         </Flex>
         <TextContent>
-          <Text>{t.rich("consumer_name", { consumerGroupName })}</Text>
+          <Text>{t.rich("consumer_name", { groupId: consumerGroupName })}</Text>
         </TextContent>
```

Comment:
This seems redundant now that it's in the subtitle too.
![image](https://github.com/user-attachments/assets/8049a58a-50d6-492d-b338-3eda877b57fe)

Reply:
I need to update the header on the Dryrun page. The goal is to remove the "Reset Offset" header and replace it with "Dryrun" as the main page header. To achieve this, I'll need to modify the routing configuration to ensure the correct header is displayed. I'll start working on this change shortly.

Reply:
I will merge this PR, will fix the "Reset Offset" header issue in the separate PR